LICENSE

```diff
@@ -199,3 +199,39 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+========================================================================
+Apache 2.0 licenses
+========================================================================
+
+The following components are provided under the Apache License. See project link for details.
+The text of each license is the standard Apache 2.0 license.
+   Google: guava 28.1: https://github.com/google/guava , Apache 2.0
+   Spring Framework 4.3.14.RELEASE: https://github.com/spring-projects/spring-framework, Apache 2.0
+   Spring Boot 1.5.10: https://spring.io/, Apache 2.0
+   Apache: log4j2 2.6.2: https://github.com/apache/logging-log4j2, Apache 2.0
+   netty 4.1.63: https://github.com/netty/netty/blob/4.1/LICENSE.txt, Apache 2.0
+   flatbuffers files from istio/proxy: https://github.com/istio/proxy Apache 2.0
```

Comment:
Why the runner requires some files from istio?

Reply:
Typo. Fixed :)